.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
```

Comment:
- Strongly recommend don't edit config files

Reply:
- [ ] **OPTIONAL**
Please keep in mind that you should always mention an explanation, like why is it important to work or do the changes you're requesting from a student! 😄 🙏